src/views/Create.vue

```diff
@@ -434,7 +423,7 @@ export default {
 			top: var(--header-height);
 			display: flex;
 			align-items: center;
-			align-self: flex-end;
+			left: 44px;
```

Comment:
Any reasons for this? What were you trying to do here?

Reply:
It’s moving the "Add question" menu to the left instead of to the right, and always leaving 44px for the navigation toggle.

Reply:
@skjnldsv Are you fine with this?

Reply:
No, please use the same syntax that was here before:
```suggestion
			align-self: flex-start
```

Reply:
It's done. If you're good, you can merge...